features/conversation/conversation-create/conversation-create-list-results.tsx

```diff
@@ -245,18 +246,44 @@ export const ConversationCreateListResults = memo(function ConversationCreateLis
   const { data: unstoppableDomainEthAddressResolution } =
     useUnstoppableDomainNameResolution(searchTextValue)
 
+  // Fetch DM data to get peerInboxIds
+  const dmsWithPeerInboxIds = useMemo(() => {
+    const result: { xmtpConversationId: IXmtpConversationId; peerInboxId?: IXmtpInboxId }[] = []
+    
+    for (const xmtpConversationId of existingDm) {
+      const dmData = reactQueryClient.getQueryData(
+        getDmQueryOptions({
+          clientInboxId: currentUserInboxId,
+          xmtpConversationId,
+          caller: "ConversationCreateListResults",
+        }).queryKey
+      )
+      
+      if (dmData?.peerInboxId) {
+        result.push({
+          xmtpConversationId,
+          peerInboxId: dmData.peerInboxId,
+        })
+      } else {
+        result.push({ xmtpConversationId })
+      }
+    }
+    
+    return result
+  }, [existingDm, currentUserInboxId])
+
```

Comment:
why this? can't we just use `getConversationQueryData(...)`  where we actually need the peerInboxId?